lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -0,0 +1,104 @@
+module Fastlane
+    module Actions
+      class IosLintLocalizationsAction < Action
+        def self.run(params)
+          UI.message "Linting localizations for parameter placeholders consistency..."
+    
+          require_relative '../../helper/ios/ios_l10n_helper.rb'
+          helper = Fastlane::Helpers::IosL10nHelper.new(
+            install_path: resolve_path(params[:install_path]),
+            version: params[:version]
+          )
+          violations = helper.run(
+            input_dir: resolve_path(params[:input_dir]),
+            base_lang: params[:base_lang],
+          )
+        
+          violations.each do |lang, diff|
+            UI.error "Inconsistencies found between '#{params[:base_lang]}' and '#{lang}':\n\n#{diff}\n"
+          end
+          UI.abort_with_message!('Inconsistencies found during Localization linting. Aborting.') unless violations.empty?
```

Comment:
[Open question]
I considered that localization violations should always be errors and make the lane abort early (as opposed to only be a warning) because those violations will make the app crash in other locales if not fixed; but I'm open to make them a warning instead if you think that would fit our sprint workflow better.

That also might depend on where we want to call this action from in our app's `Fastfiles` (I'm thinking theses would be called during `complete_code_freeze` and should prevent the code freeze from going forward if there are still violations to be fixed, but maybe you have other typical call sites in mind?

Reply:
I never had to deal with localization issues, so I don't have a solid experience on which to base my suggestion.

I like the idea of failing early 👍 

The worst thing that I can see happening is the release being delayed because there's a broken string that we release managers don't know how to fix and we need to wait for the translators' input. That's better than crashing on the user 😄 

Plus, maybe we could have a check one or two days before the time to finalize the release to see how the strings that have already been localized pass the lint.

Reply:
I totally agree that localization violations should be errors.

Reply:
Pushed a new commit to make it abort by default but allow to opt-out of abort behavior via an optional `abort_on_violations` param.

I think we'll definitively make it error and abort during `finalize_release` (and not during `complete_code_freeze` like I initially said in my PR description, sorry), but I'm still wondering if we should also make it errors for `new_beta_release` (probably yes too, but open for debate)

Reply:
+1 for `new_beta_release`, too.

The opt-out parameter is great. Gives us the best of both world. If the lint fails and we have the tool to fix it, we can address it. If it fails but we still need to push a build out, e.g. to test an important bug fix, we can skip.

Excellent 👍